var/spack/repos/builtin/packages/py-pyliblzma/package.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPyliblzma(PythonPackage):
+    """Python bindings for liblzma"""
```

Comment:
Should this also depend on `lzma`? Often when there is a Python bindings package it makes sense for that package to depend on the library it is providing bindings for.

Reply:
I was also thinking about that. The package installed just fine, so I wasn't sure. But, It makes sense.